### DISCUSSION TITLE
Should parameters inherit OrderedDict?

### DISCUSSION THREAD

**Discussion by marcosfelt** [lmfit/lmfit-py#818] [General]
Currently, `Parameter` inherits from `dict` instead of `OrderedDict`. Therefore, calling `values` is not guaranteed to return the parameter values in the same order every time. Am I missing something here?

https://github.com/lmfit/lmfit-py/blob/4173760ee020e5707e0dda5e604145f11b8659b2/lmfit/parameter.py#L25

**Comment by reneeotten** [comment]
the order in dictionaries is preserved since Python 3.7. 

https://mail.python.org/pipermail/python-dev/2017-December/151283.html

I'm sure it's described in the release notes as well but I cannot find that easily right now. 

https://github.com/lmfit/lmfit-py/pull/713

**Reply by marcosfelt** [reply]
Thank you!
